bigquery/samples/copy_table_cmek.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def copy_table_cmek(client, dest_table_id, orig_table_id):
+
+    # [START bigquery_copy_table_cmek]
+    from google.cloud import bigquery
+
+    # TODO(developer): Construct a BigQuery client object.
+    # client = bigquery.Client()
+
+    # TODO(developer): Set dest_table_id to the ID of the destination table.
+    # dest_table_id = "your-project.your_dataset.your_table_name"
+
+    # TODO(developer): Set orig_table_id to the ID of the original table.
+    # orig_table_id = "your-project.your_dataset.your_table_name"
+
+    # Set the encryption key to use for the destination.
+    # TODO(developer): Replace this key with a key you have created in KMS.
+    kms_key_name = "projects/{}/locations/{}/keyRings/{}/cryptoKeys/{}".format(
+        "cloud-samples-tests", "us", "test", "test"
+    )
```

Comment:
Let's change this into a comment and pass in the `kms_key_name` like we do table IDs.

```suggestion
    # kms_key_name = "projects/your-project/locations/us/keyRings/your-ring/cryptoKeys/your-ring"
```